<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>谱面制作器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/nku.png</normaloff>:/nku.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item>
       <widget class="QPushButton" name="ChooseMusBtn">
        <property name="text">
         <string>选择音乐</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PauseMusBtn">
        <property name="text">
         <string>暂停播放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PlayMusBtn">
        <property name="text">
         <string>播放音乐</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="BPMLabel">
        <property name="text">
         <string>输入歌曲BPM：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="BPMInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="BPMSetBtn">
        <property name="text">
         <string>设置BPM</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="TimeRemainShow">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StartBtn">
        <property name="text">
         <string>开始打谱</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PauseBtn">
        <property name="text">
         <string>暂停打谱</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ResumeBtn">
        <property name="text">
         <string>继续打谱</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StopBtn">
        <property name="text">
         <string>停止打谱</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ReStartBtn">
        <property name="text">
         <string>重新开始</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>未开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label3">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>未开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OutBtn">
        <property name="text">
         <string>导出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
